### PR TITLE
Login Screen and Service

### LINKED ISSUE #1: Create login screen
This is quite easy and self-explanatory.
Beaker Server is able to authenticate the upcoming users. Therefore we can implement the following part
- New Login screen
- BasicAuth mechanism, Beaker returns cookies on successful auth.
- Showing identity on the main bar.
- Login Service

https://beaker-project.github.io/beaker-next-design/designs/login/

### PR DESCRIPTION
**Pull request by renanrodrigo**
Implement the initial login screen and data flow to make it work.

As this is the first screen implemented, some commits are changing locations and standards to make it more organized.

Fix #1

### CODE REVIEW

**Comment by renanrodrigo** [comment]
@mdujava if you want to take a look as well

**Review by StykMartin — APPROVED** [review]
🐬

File: src/services/api.ts [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import axios from 'axios';
 
+import config from '../app.config';
```
Comment (StykMartin) [inline src/services/api.ts:3]: `@services`?

Comment (renanrodrigo) [reply src/services/api.ts]: `@config`... missed it

**Pull request merged**